conftest.py

```diff
@@ -3,6 +3,7 @@
 import tempfile
 from importlib.util import find_spec
 from pathlib import Path
+from sys import version_info as py_version_info
```

Comment:
To respect uniformity across the code base
```suggestion
import sys
```